.github/workflows/build.yml

```diff
@@ -84,3 +84,44 @@ jobs:
         push: ${{ github.event_name != 'pull_request' }}
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+
+  create_phar:
+    needs: build
+    runs-on: ubuntu-latest
+
+    if: github.event_name == 'release'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
```

Comment:
I suggest to change this to php 7.1, since 7.1 is the lowest version we decided to support. How does that sound?

Reply:
I got problems with the build with 7.1 but I could retry to understand well the problem.

Reply:
@micheleorselli this is the reason:
https://github.com/phparkitect/arkitect/runs/2498698001?check_suite_focus=true

Have you ever seen that error?

Reply:
It seems fixed with this additional parameter into the configuration:

`"dump-autoload": false`